frame/elections-phragmen/src/lib.rs

```diff
@@ -784,6 +784,18 @@ impl<T: Trait> Contains<T::AccountId> for Module<T> {
 		Self::is_member(who)
 	}
 	fn sorted_members() -> Vec<T::AccountId> { Self::members_ids() }
+
+	#[cfg(feature = "runtime-benchmarks")]
+	// A special function to populate members in this pallet for passing Origin
+	// checks in runtime benchmarking.
```

Comment:
insubstantial but in the codebase I think comment always go first, this is to avoid having the attributes far from the function signature in case the comments get very long.

```suggestion
	// A special function to populate members in this pallet for passing Origin
	// checks in runtime benchmarking.
	#[cfg(feature = "runtime-benchmarks")]
```